### DISCUSSION TITLE
Hex, RGB, or RGBA colours for Qt and GTK clients

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#5800] [Ideas]
Starting with the discussion in #5648, i've been trying to unify the main up/down/highlight colours used across the different clients. It's been rougher than expected. 

#5762 will give parity between macOS and the web client, but i've hit a wall with regard to Qt and GTK, and either need some help, but also need to make a request(s?). 

### GTK

I spent around half a day yesterday working on 4 different iterations of TorrentCellRenderer.cc 

- RGB
- RGBA
- Hex
- SVG colour name

[TorrentCellRendererIterations.txt](https://github.com/transmission/transmission/files/12063179/TorrentCellRendererIterations.txt)

Each of the variations would compile, but _none_ of them would actually change the colours used in the client. 

### Qt

Pretty much ditto with Qt, making a bunch of the same kind of iterations in TorrentDelegate.h with zero success.

**The request part**
Can i please have some easily changable hex or rgba values for the GTK and Qt clients?

GTK and Qt currently only use one colour while seeding. If possible, i'd love to see a background _and_ foreground colour like the other clients utilise.

Can the Qt client please have a separate colour for highlighting torrents?

Back in December, the Qt client had much larger progress bars (and a separate highlight colour), i'd really like to see them back so that the clients have a similar look.

![qt-ubuntu](https://user-images.githubusercontent.com/69029666/208224940-dc7008b8-f21c-49a5-b205-ebe80c1b666f.png)

**Comment by Pentaphon** [comment]
I think @livings124 may be of help with this.
